app.js

```diff
@@ -436,10 +533,12 @@ async function restartBrowser(browser) {
 }
 
 async function main() {
-  const { exec, token } = await readConfig();
+  //added game - AlexSimpler
+  const { exec, token, game} = await readConfig();
   browserConfig.executablePath = exec;
   authTokenCookie.value = token;
-
+  STREAMERS_URL = (STREAMERS_URL + game.toUpperCase();
```

Comment:
UPPER_SNAKE_CASE means it's a costant, we should parametrize the function like above

Reply:
just lowercase it then, why add complexity?

Reply:
I don't think changing a global is a good way to do it, if we added a parameter maybe later we can check for multiple game drops